data/language/en-GB.txt

```diff
@@ -3663,6 +3663,8 @@ STR_6406    :GOG RollerCoaster Tycoon 2 Installer
 STR_6407    :This may take a few minutes.
 STR_6408    :Please install “innoextract” to extract GOG Installer, then restart OpenRCT2.
 STR_6409    :The selected file is not the offline GOG Installer for RollerCoaster Tycoon 2. You may have downloaded the GOG Galaxy downloader stub or selected the wrong file.
+STR_6410    :Zoom in/out
+STR_6411    :{SMALLFONT}{BLACK}Show separate buttons for the zoom options in the toolbar
```

Comment:
```suggestion
STR_6411    :{SMALLFONT}{BLACK}Show buttons for zooming in and out in the toolbar
```

"Separate" implies that turning the option off will merge the buttons into one, rather than hide them.

Reply:
Ah, right. I just matched the wording of the other tooltips in that group when I first created this, hadn't considered that yet.
Can I also drop `{SMALLFONT}{BLACK}` from the string or does that require additional work? (looking at \#13560)

Reply:
Yeah, you can drop that. Good point, forgot about that.